### PR TITLE
Add logo to README, add workaround for GitVersion to produce a versio…

### PR DESCRIPTION
**Pull request by jhindulak**
* Add a logo to the README
* Add a workaround for GitVersion so that it can properly generate a version when using `Mainline` development mode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Trebuchet - Launch container images into Amazon ECR
 [![Build Status](https://travis-ci.org/HylandSoftware/trebuchet.svg?branch=master)](https://travis-ci.org/HylandSoftware/trebuchet) [![Coverage Status](https://coveralls.io/repos/github/HylandSoftware/trebuchet/badge.svg?branch=master)](https://coveralls.io/github/HylandSoftware/trebuchet?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/hylandsoftware/spot)](https://goreportcard.com/report/github.com/hylandsoftware/trebuchet)
 
+<img src="https://github.com/HylandSoftware/trebuchet/raw/master/logo/trebuchet_640x640.png" width="200">
```
Comment (nlowe) [inline README.md]: Let's use a markdown link here instead of HTML:

```suggestion
![trebuchet logo](./logo/trebuchet_640x640.png =200x200)
```

**Comment by MattLuceen** [comment]
What's the `treb_640x640.png` for?

**Comment by mfornaro** [comment]
> What's the `treb_640x640.png` for?

Two logos, one with full name trebuchet, one with shortened name of treb.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Trebuchet - Launch container images into Amazon ECR
 [![Build Status](https://travis-ci.org/HylandSoftware/trebuchet.svg?branch=master)](https://travis-ci.org/HylandSoftware/trebuchet) [![Coverage Status](https://coveralls.io/repos/github/HylandSoftware/trebuchet/badge.svg?branch=master)](https://coveralls.io/github/HylandSoftware/trebuchet?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/hylandsoftware/spot)](https://goreportcard.com/report/github.com/hylandsoftware/trebuchet)
 
+![](./logo/trebuchet_640x640.png =200x200)
```
Comment (nlowe) [inline README.md]: Damn, I could have sworn github supported this, I know mattermost does. Sorry about that. If you really want it to be 200x200 you'll have to use the img tag you had before.

Comment (jhindulak) [reply README.md]: I was able to get a 200x200 image, so I'm replacing the markdown with that.

File: README.md (hunk 1 above)
Comment (jhindulak) [reply README.md]: Replaced with 200x200 size icon.

**Pull request merged**